### PR TITLE
[Enhancement] optimize delta2bboxes

### PR DESCRIPTION
**Pull request by grimoire**
Optimize delta2bbox according to latest mmdetection.
remove expand, less nodes generated.

old:
Screenshot from 2022-02-12 17-08-24

new:
Screenshot from 2022-02-12 17-08-32

### CODE REVIEW

File: mmdeploy/codebase/mmdet/deploy/utils.py [hunk 1]
```diff
@@ -44,6 +44,7 @@ def clip_bboxes(x1: Tensor, y1: Tensor, x2: Tensor, y2: Tensor,
     """
     assert len(max_shape) == 2, '`max_shape` should be [h, w]'
     if isinstance(max_shape, torch.Tensor):
+        # TODO: This is only required for TensorRT 7.x
```
Comment (RunningLeon) [inline mmdeploy/codebase/mmdet/deploy/utils.py:47]: could remove this comment.

File: mmdeploy/codebase/mmdet/core/bbox/delta_xywh_bbox_coder.py [hunk 2]
```diff
@@ -92,48 +92,42 @@ def delta2bbox(ctx,
...
-    means = deltas.new_tensor(means).view(1,
-                                          -1).repeat(1,
-                                                     deltas.size(-1) // 4)
-    stds = deltas.new_tensor(stds).view(1, -1).repeat(1, deltas.size(-1) // 4)
-    denorm_deltas = deltas * stds + means
-    dx = denorm_deltas[..., 0::4]
-    dy = denorm_deltas[..., 1::4]
-    dw = denorm_deltas[..., 2::4]
-    dh = denorm_deltas[..., 3::4]
+    means = deltas.new_tensor(means).view(1, -1)
```
Comment (RunningLeon) [inline mmdeploy/codebase/mmdet/core/bbox/delta_xywh_bbox_coder.py:95]: could we update for `delta2bbox__ncnn` as well?

**Review by RunningLeon — APPROVED** [review]
LGTM

**Review by hanrui1sensetime — APPROVED** [review]
LGTM.

**Pull request merged**